util/DEPS.py

```diff
@@ -0,0 +1,24 @@
+# If you're familiar with Chromium or other Google projects:
+#   Sorry, this file actually has nothing to do with gclient.
+#   I just borrowed the basic concept and naming scheme.
+
+deps = {
+    'ACT': {
+        'url': 'https://github.com/EQAditu/AdvancedCombatTracker/releases/download/3.4.5.265/ACTv3.zip',
+        'dest': 'plugin/ThirdParty/ACT',
+        'strip': 0,
+        'hash': ['sha256', 'adf13a38d0938ce90f8e674f8365b227d933b91636ddf72b26c85702f6e3b808'],
```

Comment:
What is this a hash of, and how do I update it when the version changes?

Reply:
The hash is generated from the file content. It's just to make sure the downloaded file is what we expect.
You can update the hash by running the `fetch_deps.py` script with the `--update-hashes` option. The logic is a bit hacky (it does a simple string replace and replaces the old hash with the new hash). If you were to use JSON instead of Python as the format for the `DEPS.py` file (and rename it to `DEPS.json` or whatever), this could be improved (since you can just serialize the modified object again).